recipes/marvin/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "marvin" %}
+{% set version = "2.3.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f96628c5e7292ef02cf4bc1f4e6692df23cd1622e4f8c7cfcb2190bda38619d6
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python 3.9
```

Comment:
I think you need to use the `python_min` convention mentioned in https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python for new `noarch` packages. I tend to use the jinja variable `{% set python_min = "3.9" %}` explicitly at the top of the recipe.